features/bevragen/persoon-beperkt/verblijfplaats/dev/fields-gba.feature

```diff
@@ -0,0 +1,394 @@
+    | fields                                |
+    | verblijfplaatsBinnenland              |
+    | verblijfplaatsBinnenland.functieAdres |
+    | verblijfplaatsBinnenland.type         |
+
+Rule: als één of meerdere (gegevensgroep) velden van een verblijfplaats wordt gevraagd en de categorie verblijfplaats (08) heeft in onderzoek gegevens, dan worden deze ook geleverd
+
+  Abstract Scenario: '<type>' van een adres is in onderzoek en één of meerdere velden wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde |
+    | gemeentecode (92.10) | 0519   |
+    | straatnaam (11.10)   | Spui   |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <fields>                        |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                      | waarde                    |
+    | straat                                    | Spui                      |
+    | inOnderzoek.aanduidingGegevensInOnderzoek | <aanduiding in onderzoek> |
+    | inOnderzoek.datumIngangOnderzoek          | 20020701                  |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | fields                                                 | type                          |
+    | 080000                  | verblijfplaats                                         | hele categorie verblijfplaats |
+    | 081100                  | verblijfplaats.functieAdres                            | hele groep adres              |
+    | 081150                  | verblijfplaats.type                                    | aanduiding bij huisnummer     |
+    | 081130                  | verblijfplaats.functieAdres,verblijfplaats.inOnderzoek | huisletter                    |
+    | 081160                  | verblijfplaats                                         | postcode                      |
+    | 081020                  | verblijfplaats.type                                    | gemeentedeel                  |
+    | 080910                  | verblijfplaats                                         | gemeente van inschrijving     |
+    | 081410                  | verblijfplaats.functieAdres                            | land vanwaar ingeschreven     |
+
```

Comment:
Ben even kwijt waarom "land vanwaar ingeschreven in onderzoek" relevant is voor "functie-adres in onderzoek"....